about/index.html

```diff
@@ -1,28 +1,36 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8"/>
-    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Об авторе</title>
 </head>
+
 <body>
     <h1>&#9996; Немного обо мне</h1>
-    <p>Меня зовут Оксана. Мне на данный момент 2<sup>1</sup> год. Родилась в небольшом городке и большую часть жизни прожила там. <br/>После школы я переехала в Минск и начала поиски себя. <strong>Мой девиз по жизни</strong>:</p>
+    <p>Меня зовут Оксана. Мне на данный момент 2<sup>1</sup> год. Родилась в небольшом городке и большую часть жизни
+        прожила там. <br />После школы я переехала в Минск и начала поиски себя. <strong>Мой девиз по жизни</strong>:
```

Comment:
We have discussed `br` in another inline tag.

Reply:
replace tag br in tag p